modules/ROOT/pages/errors/all-errors.adoc

```diff
@@ -4,16 +4,16 @@
 [[neo4j-errors]]
 = List of all error codes
 
-This page contains lists of all Neo4j errors, grouped by class.
+This page contains lists of all Neo4j errors, grouped by type.
 
 == Client errors
 
-This is a complete list of all client error status codes Neo4j may return, and what they mean.
+This is a complete list of all client error codes Neo4j may return, and what they mean.
```

Comment:
I think it is OK if we like to use a separate wording than `Neo4j status code` for errors. But now there is a mix of e.g. `[client] error codes`, `[transient] error status code`, `Neo4j error code`. I think we should try to be more consistent. 

Also if we not decide to call it just `Neo4j status codes`, I think we should add a note to clarify that it is a subset of `Neo4j status codes`